jenkins-agent-k8s/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM jenkins/jnlp-slave:3.23-1-alpine
```

Comment:
`3.27-1 ` is the latest version

Reply:
Since you did the upstream image, I would appreciate if you just apply the fixes to this branch. Thanks :)